packages/app-bridge/src/react/useGuidelineAssetsSettings.ts

```diff
@@ -0,0 +1,27 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect, useState } from 'react';
+import { Handler } from 'mitt';
+
+import type { GuidelineAssetsSettings } from '../types/GuidelineAssetsSettings';
+
+export const useGuidelineAssetsSettings = () => {
+    const [assetsSettings, setAssetsSettings] = useState<GuidelineAssetsSettings>({
+        imageAndVideoDownloadEnabled:
+            window.application?.sandbox?.config?.context?.appearance?.privacy?.image.download === 'enable',
+        assetViewerEnabled: !!window.application?.sandbox?.config?.context?.appearance?.asset_viewer_enabled,
```

Comment:
Move the "getter" to app bridge, the react hook shouldn't know about the implementation of how we get the data